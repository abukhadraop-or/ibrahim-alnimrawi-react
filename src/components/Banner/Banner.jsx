import { BannerDiv, BannerH1, BannerP } from "./banner.style";
import React from "react";
const Banner = () => {
  return (
    <BannerDiv>
      <BannerH1>conduit</BannerH1>
      <BannerP>A place to share your knowledge.</BannerP>
    </BannerDiv>
  );
};

export default Banner;
