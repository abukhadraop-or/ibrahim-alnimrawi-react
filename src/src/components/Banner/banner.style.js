import styled from "styled-components";
import colors from "../../services/colors";
export const BannerDiv = styled.div`
  align-items: center;
  background-color: ${colors.moderateLimeGreen};
  border: none;
  box-shadow: inset 0 0.5rem 0.5rem -0.5rem rgb(0 0 0 / 30%),
    inset 0 -0.5rem 0.5rem -0.5rem rgb(0 0 0 / 30%);
  color: ${colors.white};
  display: flex;
  flex-direction: column;
  margin-bottom: 2rem;
  padding-top: 0;
  padding: 1rem;
`;
export const BannerH1 = styled.h1`
  font-size: 3.5rem;
  font-weight: 700;
  margin-bottom: 0.5rem;
  text-shadow: 0 0.063rem 0.188rem rgb(0 0 0 / 30%);
`;
export const BannerP = styled.p`
  font-size: 1.5rem;
  font-weight: 300;
  margin-bottom: 0;
  text-align: center;
`;
