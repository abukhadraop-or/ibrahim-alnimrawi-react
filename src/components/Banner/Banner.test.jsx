import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Banner from 'components/Banner/Banner';
import Enzyme from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

Enzyme.configure({ adapter: new Adapter() });

describe('Banner component', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Banner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
