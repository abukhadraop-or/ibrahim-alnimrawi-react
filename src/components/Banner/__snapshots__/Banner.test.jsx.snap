// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner component should match snapshot 1`] = `
<div
  className="sc-bdfBQB gTaVek"
>
  <h1
    className="sc-gsTEea elcpli"
  >
    conduit
  </h1>
  <p
    className="sc-dlfnuX yRSye"
  >
    A place to share your knowledge.
  </p>
</div>
`;
